src/Client.php

```diff
@@ -554,11 +592,27 @@ public function request(string $path, string $method = Request::GET, $data = [],
             return $this->request($path, $method, $data, $query);
         }
 
-        $this->_logger->debug('Elastica Request', [
-            'request' => $request->toArray(),
-            'response' => $response->getData(),
-            'responseStatus' => $response->getStatus(),
-        ]);
+        if ($this->shouldLog()) {
+            $elapsedTimeMs = (int)(round($response->getQueryTime() * 1000));
```

Comment:
Maybe extract to separate method logRequest?

Reply:
I will if this method gets beefier. :+1: